README.md

```diff
@@ -43,6 +43,18 @@ withCredentials([usernamePassword(credentialsId: 'amazon', usernameVariable: 'US
 }
 ```
 
+#### Note
+
+You should use a single quote (') instead of a double quote (") whenever you can. 
```

Comment:
```suggestion
You should use a single quote (') instead of a double quote (") whenever you can to provide data to pipeline steps. 
```

Reply:
I don't know. Isn't the current statement correct? -- You should use a single quote whenever you can. Does qualifying it like help the user understand better?

Reply:
Oh I didn't think of it that way; good point. Seems fine the way you wrote it then.